openc3-traefik/Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG OPENC3_DEPENDENCY_REGISTRY=docker.io
-FROM ${OPENC3_DEPENDENCY_REGISTRY}/traefik:2.9.10
+FROM ${OPENC3_DEPENDENCY_REGISTRY}/traefik:2.10.4
```

Comment:
Upgrade release notes: https://doc.traefik.io/traefik/v2.10/migration/v2/#v210
Looks like Kubernetes customers need to do this before upgrading (will need to add to release notes):
```
kubectl apply -f https://raw.githubusercontent.com/traefik/traefik/v2.10/docs/content/reference/dynamic-configuration/kubernetes-crd-rbac.yml
kubectl apply -f https://raw.githubusercontent.com/traefik/traefik/v2.10/docs/content/reference/dynamic-configuration/kubernetes-crd-definition-v1.yml
```